src/Analyzers/VisualBasic/Analyzers/SimplifyObjectCreation/VisualBasicSimplifyObjectCreationDiagnosticAnalyzer.vb

```diff
@@ -0,0 +1,53 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.CodeStyle
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.SimplifyObjectCreation
+    <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
+    Friend NotInheritable Class VisualBasicSimplifyObjectCreationDiagnosticAnalyzer
+        Inherits AbstractBuiltInCodeStyleDiagnosticAnalyzer
+
+        Public Sub New()
+            MyBase.New(
+                diagnosticId:=IDEDiagnosticIds.SimplifyObjectCreationDiagnosticId,
+                enforceOnBuild:=EnforceOnBuildValues.SimplifyObjectCreation,
+                [option]:=Nothing,
```

Comment:
This should have an option

Reply:
The day where this isn't needed anymore can't get here soon enough

Reply:
For the option to be useful, it needs to be involved in code generation. Otherwise, setting severity should be enough. Anything I'm missing?

Reply:
yeah, we need an option so that people can configure this from the UI.  for example, stating that they want this, and what severity this should be.

Reply:
@CyrusNajmabadi I think @jmarolf's PR (https://github.com/dotnet/roslyn/pull/51069) will solve that in a more global and cleaner way.